elements/pl-matching/pl-matching.py

```diff
@@ -0,0 +1,399 @@
+import prairielearn as pl
+import lxml.html
+import random
+import math
+import chevron
+
+WEIGHT_DEFAULT = 1
+FIXED_ANSWER_ORDER_DEFAULT = False
+INLINE_DEFAULT = False
+PARTIAL_CREDIT_DEFAULT = True
+HIDE_ANSWER_PANEL_DEFAULT = False
+HIDE_HELP_TEXT_DEFAULT = False
+DETAILED_HELP_TEXT_DEFAULT = False
+HIDE_LETTER_KEYS_DEFAULT = False
+HIDE_SCORE_BADGE_DEFAULT = False
+BLANK_DEFAULT = False
+BLANK_ANSWER = ' '
+NOTA_DEFAULT = False
+COUNTER_TYPE_DEFAULT = 'lower-alpha'
+
+
+def get_form_name(answers_name, index):
+    return f'{answers_name}-dropdown-{index}'
+
+
+def get_counter(i, counter_type):
+    """Converts an integer counter to the specified CSS counter type"""
+    if counter_type == 'lower-alpha':
+        return chr(ord('a') + i - 1)
+    elif counter_type == 'upper-alpha':
+        return chr(ord('A') + i - 1)
```

Comment:
A bit of a pet-peeve, but this could be a problem if there are more than 26 options...

Reply:
Definitely hacky. I wasn't aware that CSS goes from `z` to `aa`, `ab`, etc; will change to handle this.

Reply:
Wrote a decimal-to-base-26 helper.